go.mod

```diff
@@ -26,7 +26,7 @@ require (
 	github.com/boltdb/bolt v1.3.1 // indirect
 	github.com/clusterhq/flocker-go v0.0.0-20160920122132-2b8b7259d313
 	github.com/container-storage-interface/spec v1.3.0
-	github.com/containernetworking/cni v0.8.0
+	github.com/containernetworking/cni v0.8.1
```

Comment:
last release is 0.9.1 https://github.com/containernetworking/plugins/releases/tag/v0.9.1

Reply:
A quick question here:

Quota from https://github.com/containernetworking/cni/releases/tag/v0.8.0
> We do not release binaries here: this repo holds the Specification and Go Library for CNI; the plugins have moved to https://github.com/containernetworking/plugins.

What's the relationship between two repos: https://github.com/containernetworking/cni and https://github.com/containernetworking/plugins? And cni v0.8.1 and plugins v0.9.1 released on a similar date.

- https://github.com/containernetworking/plugins/releases/tag/v0.9.1 on Feb 5th.
- https://github.com/containernetworking/cni/releases/tag/v0.8.1 on Feb 2th.

Reply:
We're only using the libs/types from repo `github.com/containernetworking/cni` (plugins repo is not vendored).

Reply:
yeah, my bad , I mixed both of them 0.8.1 is the right one